source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,129 @@
+It covers the actions required by the Data Catalogue team in the event of a disaster, focusing on minimizing downtime and data loss.
+
+## Summary
+
+- Find MoJ data is a Django application that calls a backend service [DataHub] for all of its data. Both services are hosted on the MoJ's [Cloud Platform] (CP), and use CP for integrated services like the application database ([AWS RDS PostgreSQL]) and search service (AWS OpenSearch).
+
+- Both Find MoJ data and DataHub deployments have RDS PostgreSQL databases deployed via CP and so get [automated daily backups] and the last 7 are retained.
+
+  > **Note**: due to an implementation bug, as of 2024-09-03 the snapshot backups are taken _**every other morning**_, meaning the oldest backup is 12-13 days old, and that the most recent backup can be up to 48 hours old.
+
+- Following loss or corruption of database data, we can restore from one of these backups by [following CP's RDS snapshot restore guidance].
+
+  - This process should be tested every 6-12 months to ensure the team is familiar with the process, and that there are no unexpected issues.
+
+  - The Find MoJ data RDS database contains app usage data, which would not be overly detrimental to operation of the service if it were to be lost.
+
+  - The DataHub RDS database contains catalogue metadata aggregated from other data-handling services along with application data (user data, usage data). _(As of 2024-09-03)_ The current aims for our DataHub are to aggregate metadata as-is from data sources, so total loss of this metadata would not result in loss of information for the business, as it could all be re-ingested into the catalogue application.
+
+- The Find MoJ data service is not considered to be a critical service, and as such we have single RDS instances for each of our deployed environments. We do not have multi-Availability Zone (AZ) deployments and/or read replicas for our databases, as the business will be able to tolerate reasonable outages due to aws-level disruptions (e.g. AZ- or region-level outages).
+
+- In the medium term, we do not expect more than 50 simultaneous users for any length of time. At full operation, we could expect to see hundred-to-thousands of unique monthly users, and at this time we may need to re-visit our recovery objectives.
+
+## Scope
+- **Application**: Django-based Python application
+- **Backend services**: DataHub
+- **Hosting**: MoJ's Cloud Platform: AWS EKS
+- **Database**: AWS RDS PostgreSQL
+- **Key Components**:
+  - Django application code
+  - DataHub application
+  - AWS RDS PostgreSQL database
+  - Docker images stored in Amazon ECR
+
+## Business appetite for time to recovery
+- Our service has not yet reached full deployment within the organisation, and there are no services dependant upon its function. As such, the service could become unresponsive for extended periods without compromising end-user deliver objectives.
```

Comment:
I'm not parsing this last sentence - do you mean delivery objectives of people consuming the data?

Reply:
I've adjusted this sentence, hopefully clearer now. It was referring to anyone wanting to use our service.